src/main/java/org/candlepin/testext/hostedtest/HostedTestCloudRegistrationAdapter.java

```diff
@@ -32,6 +32,8 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import org.apache.commons.codec.binary.Base64;
```

Comment:
Since Java 8 the java.util.Base64 should be preferred over the one from apache.commons.